tests/unit/test_modify.py

```diff
@@ -0,0 +1,111 @@
+# coding: utf-8
+
+from __future__ import unicode_literals, absolute_import
+
+import unittest
+import os
+import tempfile
```

Comment:
Since the testsuite is using pytest one can use https://docs.pytest.org/en/stable/tmpdir.html#the-tmpdir-fixture
However it should be OK to keep as it is now.